test/resources/logback-test.xml

```diff
@@ -6,6 +6,8 @@
         </encoder>
     </appender>
 
+    <logger name="application" level="OFF"/>
```

Comment:
This is just for turning off the unit test logging noise?

Reply:
Yup, that exactly